constellations/orion/src/tests_integration/account_integration_test.go

```diff
@@ -72,13 +71,52 @@ func TestGetAccountById(t *testing.T) {
 	assert.EqualValues(t, "john_smith@example.com", account.PrimaryEmail)
 	assert.EqualValues(t, "password1", account.Password)
 
+	utils.ResetTable(t, domains.TABLE_USERS)
+	utils.ResetTable(t, domains.TABLE_ACCOUNTS)
+}
+
+// Test: Create 2 accounts with the same email
+func Test_CreateSameEmailAccountsFailure(t *testing.T) {
+	accountUser := domains.AccountUser{
+		Account: createAccount(1),
+		User:    createUser(1),
+	}
+	body := utils.CreateJsonBody(&accountUser)
+	recorder1 := utils.SendHttpRequest(t, http.MethodPost, "/api/accounts/create", body)
+	assert.EqualValues(t, http.StatusOK, recorder1.Code)
+
+	// Create account with same email
+	accountUser2 := domains.AccountUser{
+		Account: createAccount(4),
+		User:    createUser(1),
+	}
+	body2 := utils.CreateJsonBody(&accountUser2)
+	recorder2 := utils.SendHttpRequest(t, http.MethodPost, "/api/accounts/create", body2)
+	assert.EqualValues(t, http.StatusNotImplemented, recorder2.Code)
+
+	utils.ResetTable(t, domains.TABLE_USERS)
+	utils.ResetTable(t, domains.TABLE_ACCOUNTS)
+}
+
+// Test: Create account with user
+func Test_CreateAccountAndUserSuccess(t *testing.T) {
+	accountUser := domains.AccountUser{
+		Account: createAccount(1),
+		User:    createUser(1),
+	}
+	body := utils.CreateJsonBody(&accountUser)
+
+	recorder1 := utils.SendHttpRequest(t, http.MethodPost, "/api/accounts/create", body)
+	assert.EqualValues(t, http.StatusOK, recorder1.Code)
+
+	utils.ResetTable(t, domains.TABLE_USERS)
 	utils.ResetTable(t, domains.TABLE_ACCOUNTS)
 }
 
 // Test: Create 3 Accounts, GetNegativeBalances()
 func Test_GetNegativeBalanceAccounts(t *testing.T) {
```

Comment:
Looks like I forgot to remove _ from integration test names. Can you remove them in this file?
`TestGetNegativeBalanceAccounts`